client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -0,0 +1,96 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isFreeJetpackPlan } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class DisconnectSurvey extends Component {
+	state = {
+		reasonSelected: 'onlyNeedFree',
+		renderInitial: true,
+	};
+
+	renderFollowUp() {
+		// placeholder
+		return <Card className="disconnect-site__question">{ 'follow-up' }</Card>;
+	}
+
+	logSelection = option => {
```

Comment:
`logSelection` confuses me a bit. I expected this to write to the console, or register something in a log of some sort.

This is really an event handler, I'd prefer a name that clarifies that such as `handleSelection` or even `handleQuestionClick`.

Reply:
`handleAnswerClick`, no? 😉